modules/manage/pages/cluster-maintenance/manage-throughput.adoc

```diff
@@ -43,48 +45,198 @@ The properties for broker-wide throughput quota balancing are configured at the
+
+Redpanda supports the following Kafka API-based quota types on clients:
+
+|===
+| Quota type | Description
 
-NOTE: The client throughput quotas limit the rates within each shard (logical CPU core) of a Redpanda broker's node. The quotas are neither shared nor balanced between shards and brokers.
+| `producer_byte_rate`
+| Limit throughput of produce requests
+
+| `consumer_byte_rate`
+| Limit throughput of fetch requests
+
+| `controller_mutation_rate`
+| Limit rate of topic mutation requests, including create, add, and delete partition, in number of partitions per second
+
+|===
+
+You can also apply a default quota for all other client requests that don't have a specific quota based on an exact match or `client_id` prefix. 
+
+It is possible to create conflicting quotas if you configure the same quotas through both the Kafka API and a cluster configuration. Redpanda resolves these conflicts by following an order of preference in finding a matching quota for a request:
+
+. Quota configured through the Kafka API for an exact match on `client_id`
+. Quota configured through the Kafka API for a prefix match on `client_id`
+. Quota configured through cluster configuration properties (`kafka_client_group_byte_rate_quota`, `kafka_client_group_fetch_byte_rate_quota`, xref:upgrade:deprecated/index.adoc[deprecated starting in v24.2]) for a prefix match on `client_id`
+. Default quota configured through the Kafka API on `client_id`
+. Default quota configured through cluster configuration properties (`target_quota_byte_rate`, `target_fetch_quota_byte_rate`, `kafka_admin_topic_api_rate`, xref:upgrade:deprecated/index.adoc[deprecated starting in v24.2]) on `client_id`
+
+Redpanda recommends <<migrate,migrating>> over from cluster configuration-managed quotas to Kafka-compatible quotas. You can recreate the configuration-based quotas with `rpk`, and then remove the quota cluster configs.
 
 === Individual client throughput limit
 
-To set throughput quota for a single client, configure the cluster properties xref:reference:cluster-properties.adoc#target_quota_byte_rate[`target_quota_byte_rate`] and xref:reference:cluster-properties.adoc#target_fetch_quota_byte_rate[`target_fetch_quota_byte_rate`].
+You can view current throughput quotas set through the Kafka API by running the `rpk cluster quotas describe` command.
+
+For example, to see the quotas for client ID `consumer-1`:
+
+[,bash]
+----
+rpk cluster quotas describe --name client-id=consumer-1
+----
+
+[,bash,role=no-copy]
+----
+ENTITY                QUOTA-TYPE          QUOTA-VALUE
+client-id=consumer-1  producer_byte_rate  140000
```

Comment:
@r-vasquez could you confirm if this output is correct? Since the example command doesn't include default or a user I wasn't sure if this is correct under ENTITY

Reply:
@kbatuigas No, the output is not in table format, here is the output for this example:

```
$ rpk cluster quotas describe --name client-id=consumer-1
client-id=consumer-1
	producer_byte_rate=140000
```